src/note-utils.ts

```diff
@@ -18,6 +19,8 @@ export const findNoteInPRBody = (body: string): string | null => {
     notes = notes.replace(/<!--.*?-->/g, '');
   }
 
+  debug(`Found Notes: ${JSON.stringify(notes.trim())}`);
```

Comment:
is this how `debug` works? i thought you had to create a debug function with `debug("category")` and then call that function with your log string.

Reply:
whoops, left out a line. fixed!